packages/inference/src/providers/replicate.ts

```diff
@@ -22,6 +22,7 @@ export const REPLICATE_SUPPORTED_MODEL_IDS: ProviderMapping<ReplicateId> = {
 	},
 	"text-to-speech": {
 		"OuteAI/OuteTTS-0.3-500M": "jbilcke/oute-tts:39a59319327b27327fa3095149c5a746e7f2aee18c75055c3368237a6503cd26",
+		"hexgrad/Kokoro-82M": "jaaari/kokoro-82m:dfdf537ba482b029e0a761699e6f55e9162cfd159270bfe0e44857caa5f275a6"
```

Comment:
need to add a test but you'll see, you'll need to change the param name

I was meaning to ask @jbilcke-hf to update the param name in jbilcke/oute-tts to text instead of inputs